### PR TITLE
Fix review - major changes storDir

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

I made some changes related to reviewer's comments.
the major changes are the upgrades of KOFAM and EGGNOG download modules. I used storDir to save the downloaded databases.

### CODE REVIEW

File: modules/local/eggnog/mapper.nf
Comment (erikrikarddaniel) [inline modules/local/eggnog/mapper.nf:1]: The download module had a new container, did you update this module too?

File: modules/local/kofamscan/download.nf [hunk 1]
```diff
@@ -22,15 +19,11 @@ process KOFAMSCAN_DOWNLOAD {
     def args = task.ext.args ?: ''
 
     """
-    if [ ! -e ${kofam_dir}/ko_list ]; then
-        curl https://www.genome.jp/ftp/db/kofam/ko_list.gz > ko_list.gz
-        gunzip -c ko_list.gz > ${kofam_dir}/ko_list
-    fi
+    curl https://www.genome.jp/ftp/db/kofam/ko_list.gz > ko_list.gz
+    gunzip ko_list.gz
```
Comment (erikrikarddaniel) [inline modules/local/kofamscan/download.nf]: Why are you using curl here, but wget below?

Comment (erikrikarddaniel) [inline modules/local/kofamscan/download.nf]:
```suggestion
    wget https://www.genome.jp/ftp/db/kofam/ko_list.gz
    gunzip ko_list.gz
```

File: workflows/metatdenovo.nf [hunk 2]
```diff
@@ -324,7 +324,7 @@ workflow METATDENOVO {
     //
     // Call ORFs
     //
-    ch_gff       = Channel.empty()
+    ch_gff = Channel.empty()
     ch_protein  = Channel.empty()
```
Comment (erikrikarddaniel) [inline workflows/metatdenovo.nf]: Aren't you supposed to line up the `=`s?

File: workflows/metatdenovo.nf [hunk 3]
```diff
@@ -337,7 +337,7 @@ workflow METATDENOVO {
             UNPIGZ_GFF(PROKKA_SUBSETS.out.gff.map { [ [id: "${params.orf_caller}.${it[0].id}"], it[1] ] })
             ch_versions      = ch_versions.mix(PROKKA_SUBSETS.out.versions)
             ch_gff           = UNPIGZ_GFF.out.unzipped
-            ch_protein      = PROKKA_SUBSETS.out.faa
+            ch_protein            = PROKKA_SUBSETS.out.faa
```
Comment (erikrikarddaniel) [inline workflows/metatdenovo.nf]: Similar to above, `=`s lined up?

File: workflows/metatdenovo.nf [hunk 4]
```diff
@@ -348,7 +348,7 @@ workflow METATDENOVO {
             PRODIGAL( ch_assembly_contigs.map { [ [id: "${params.assembler}.${params.orf_caller}"], it[1] ] } )
             UNPIGZ_GFF(PRODIGAL.out.gff.map { [ [id: "${it[0].id}.${params.orf_caller}"], it[1] ] })
             ch_gff          = UNPIGZ_GFF.out.unzipped
-            ch_protein      = PRODIGAL.out.faa
+            ch_protein           = PRODIGAL.out.faa
```
Comment (erikrikarddaniel) [inline workflows/metatdenovo.nf]: Again.

File: workflows/metatdenovo.nf [hunk 5]
```diff
@@ -358,7 +358,7 @@ workflow METATDENOVO {
         if ( params.orf_caller == 'transdecoder' ) {
             TRANSDECODER ( ch_assembly_contigs.map { [ [id: "transdecoder.${it[0].id}" ], it[1] ] } )
             ch_gff      = TRANSDECODER.out.gff
-            ch_protein  = TRANSDECODER.out.pep
+            ch_protein       = TRANSDECODER.out.pep
```
Comment (erikrikarddaniel) [inline workflows/metatdenovo.nf]: And again.

**Review by erikrikarddaniel — APPROVED** [review]
Download looks good. I saw a few weird indentations of `=`s.

File: modules/local/kofamscan/download.nf (hunk 1 above)
Comment (danilodileo) [reply modules/local/kofamscan/download.nf]: it was an old change and if I remember correctly the module failed with wget for that link so I used curl. I can test it again.

Comment (danilodileo) [reply modules/local/kofamscan/download.nf]: OK apparently it works with wget so I changed it!

**Pull request merged**